src/main/java/taxi/dao/impl/DriverDaoImpl.java

```diff
@@ -37,7 +40,7 @@ public Driver create(Driver driver) {
 
     @Override
     public Optional<Driver> get(Long id) {
-        String query = "SELECT * FROM drivers WHERE id = ? AND is_deleted = FALSE";
+        String query = "SELECT * FROM drivers WHERE drivers.id = ? AND is_deleted = FALSE";
```

Comment:
I think it is not a problem, but we have no join with any other table, so we don't need to specify driver.id, just id. You have the same point in other queries/

Reply:
Yeah, after the "IDEA ultimate" install, an exception was thrown for SQL language, so I fixed it automatically and this happened.

Reply:
This is very strange, we should not have any exception here without using table name before `id` field. I will not mark this as mistake now, but I recommend to show this problem on QnA and try to fix.